BDArmory/Competition/CircularSpawnStrategy.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using BDArmory.Control;
+using UnityEngine;
+using static BDArmory.Control.VesselSpawner;
+
+namespace BDArmory.Competition
+{
+    public class CircularSpawnStrategy : SpawnStrategy
+    {
+        private VesselSource vesselSource;
+        private List<int> vesselIds;
+        private double latitude;
+        private double longitude;
+        private double altitude;
+        private float radius;
+        private bool success = false;
+
+        public CircularSpawnStrategy(VesselSource vesselSource, List<int> vesselIds, double latitude, double longitude, double altitude, float radius)
```

Comment:
Not strictly using this yet, but the idea was to shift the existing circular spawn logic into here for encapsulation.